Makefile.in

```diff
@@ -178,7 +178,7 @@ HEADER_DIRS :=                                                              \
         acb_mat         acb_poly        acb_calc        acb_hypgeom         \
         arb_fmpz_poly   arb_fpwrap                                          \
         acb_dft         acb_elliptic    acb_modular     acb_dirichlet       \
-        dirichlet       bernoulli       hypgeom                             \
+        acb_theta		dirichlet       bernoulli       hypgeom             \
```

Comment:
Contains two tabs instead of spaces. Not that important. This is fixed in my commit https://github.com/albinahlback/flint/commit/807a5c3aa4115286635ef9bc000901c2dee32e11